platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/SyncGossip.java

```diff
@@ -187,7 +183,6 @@ public SyncGossip(
 
         this.threadManager = Objects.requireNonNull(threadManager);
 
-        inOrderLinker = new GossipLinker(platformContext, intakeEventCounter);
         shadowgraph = new Shadowgraph(platformContext, addressBook, intakeEventCounter);
```

Comment:
Having deleted the linker here, you may delete the `GossipLinker` class entirely, as this was the only usage

Reply:
thanks, done